src/main/java/com/adyen/model/checkout/details/AndroidPayDetails.java

```diff
@@ -33,7 +33,7 @@
  */
 
 public class AndroidPayDetails implements PaymentMethodDetails {
-    private static final String ANDROIDPAY = "androidpay";
+    public static final String ANDROIDPAY = "androidpay";
```

Comment:
Not blocking, but is this automatically generated ? It seems weird to have it public and then also have the private variable type reference, but I seem to recall that's just the way we generate it. Just want to double check.

Reply:
it's not automatically generated, we had a discussion on how to include the txvariant in the class, it was hardcoded first in the "type", and then we went for the constant approach.
In this release we are changing them from private to public, as we are comparing the type for the deserializers.